### PR TITLE
Use ansible/ansible-lint since ansible/ansible-lint-action has been archived

### PR DESCRIPTION
**Pull request by alinabuzachis**
Use ansible/ansible-lint since ansible/ansible-lint-action has been archived. (See https://github.com/ansible/ansible-lint-action)

### CODE REVIEW

**Review by hakbailey — APPROVED** [review]

**Comment by alinabuzachis** [comment]
This PR is on hold for the moment.

File: .github/workflows/sanity.yml [hunk 1]
```diff
@@ -78,6 +78,7 @@ jobs:
               "python-version": "3.9"
             }
           ]
+      collection_pre_install: '-r source/tests/sanity/requirements.yml'
```
Comment (GomathiselviS) [inline .github/workflows/sanity.yml:81]: I'm having trouble understanding the reason behind this action in the sanity workflow. Could you please clarify why this is being done and whether it addresses any specific issues or failures?

File: roles/manage_transit_gateway/README.md [hunk 2]
```diff
@@ -53,13 +49,13 @@ Example Playbook
       ansible.builtin.include_role:
         name: cloud.aws_ops.manage_transit_gateway
       vars:
-        action: "create"
-        transit_gateway:
+        manage_transit_action: "create"
```
Comment (GomathiselviS) [inline roles/manage_transit_gateway/README.md:52]:
```suggestion
        manage_transit_gateway_action: "create"
```

File: roles/manage_transit_gateway/README.md [hunk 3]
```diff
@@ -53,13 +49,13 @@ Example Playbook
...
+        manage_transit_transit_gateway:
```
Comment (GomathiselviS) [inline roles/manage_transit_gateway/README.md:53]:
```suggestion
        manage_transit_gateway_transit_gateway:
```

File: roles/manage_transit_gateway/README.md [hunk 4]
```diff
@@ -53,13 +49,13 @@ Example Playbook
...
             asn: 4200000000
             description: "TGW for Cloud team"
             tags:
               "team": "cloud"
-        vpc_attachment:
+        manage_transit_vpc_attachment:
```
Comment (GomathiselviS) [inline roles/manage_transit_gateway/README.md:58]:
```suggestion
        manage_transit_gateway_vpc_attachment:
```

File: roles/manage_transit_gateway/README.md [hunk 5]
```diff
@@ -81,13 +77,13 @@ Example Playbook
       ansible.builtin.include_role:
         name: cloud.aws_ops.manage_transit_gateway
       vars:
-        action: "create"
-        transit_gateway:
+        manage_transit_action: "create"
```
Comment (GomathiselviS) [inline roles/manage_transit_gateway/README.md:80]:
```suggestion
        manage_transit_gateway_action: "create"
```

File: roles/manage_transit_gateway/README.md [hunk 6]
```diff
@@ -81,13 +77,13 @@ Example Playbook
...
+        manage_transit_transit_gateway:
```
Comment (GomathiselviS) [inline roles/manage_transit_gateway/README.md:81]:
```suggestion
        manage_transit_gateway_transit_gateway:
```

File: roles/manage_transit_gateway/README.md [hunk 7]
```diff
@@ -81,13 +77,13 @@ Example Playbook
...
           asn: 4200000000
           description: "TGW for Cloud team"
           tags:
             "team": "cloud"
-        vpn_attachment:
+        manage_transit_vpn_attachment:
```
Comment (GomathiselviS) [inline roles/manage_transit_gateway/README.md:86]:
```suggestion
        manage_transit_gateway_vpn_attachment:
```

File: .github/workflows/sanity.yml (hunk 1 above)
Comment (alinabuzachis) [reply .github/workflows/sanity.yml]: Removed, I needed them for a previous test, but they are not needed anymore! thanks!

**Review by abikouo — APPROVED** [review]

**Review by GomathiselviS — APPROVED** [review]

**Pull request merged**